chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,671 @@
+    "\n",
+    "$$ b | \\mathbf{x} \\sim \\mathrm{Gaussian}(\\mathbf{A}\\mathbf{x}, \\sigma^2\\mathbf{I})$$\n",
+    "\n",
+    "and in this case we specify $\\sigma^2 = 0.1$.\n",
+    "\n",
+    "$$ \\pi (b | \\mathbf{x}) = \\frac{1}{\\sqrt{(2 \\pi)^m \\sigma^{2m}}} \\mathrm{exp}\\left(-\\frac{||\\mathbf{A}\\mathbf{x}- b||^2}{2\\sigma^2}\\right) $$\n",
+    "\n",
+    "- The data distribution is the conditional distribution of $b$ given $\\mathbf{x}$.\n",
+    "- This PDF can only be evaluated for a given $\\mathbf{x}$.\n",
+    "\n",
+    "We create the data distribution object as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b = Gaussian(A@x, 0.1)\n",
+    "print(b)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before sampling or evaluating the PDF of `b`, we need to specify the value of the parameters `x`, let us choose the following value:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "particular_x = np.array([1.5, 1.5])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we condition the data distribution `b` on the given parameter `particular_x`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b_given_particular_x = b(x=particular_x)\n",
+    "print(b_given_particular_x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we have the distribution object `b_given_particular_x` that represents the data distribution given a particular value of the parameters `x`. We can now plot the PDF of this distribution:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_pdf_1D(b_given_particular_x, 1.5, 4.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can use `b_given_particular_x` to simulate noisy data assuming that the true `x` parameters is `particular_x`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b_obs = b_given_particular_x.sample()\n",
+    "print(b_obs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 1.5. The likelihood function </font> <a class=\"anchor\" id=\"r-the-likelihood-function\"></a>\n",
+    "\n",
+    "\n",
+    "We obtain the likelihood function by fixing observed data $b^\\mathrm{obs}$ in the data distribution and considering the function of $\\mathbf{x}$:\n",
+    "\n",
+    "$$L (\\mathbf{x} | b^\\mathrm{obs}) \\mathrel{\\vcenter{:}}= \\pi (b^\\mathrm{obs} | \\mathbf{x})$$\n",
+    "\n",
+    "\n",
+    "Example, given observed data  $b^\\mathrm{obs} $\n",
```

Comment:
The sentence here is not complete, something like "Since we are using a Gaussian noise model, the likelihood can be evaluated as"?

Reply:
agreed :)